app-tracking-protection/vpn-impl/src/main/res/layout/activity_report_breakage_category_single_choice.xml

```diff
@@ -66,7 +64,7 @@
                     android:inputType="none"
                     android:labelFor="@id/categoriesSelection"
                     android:maxLines="1"
-                    android:paddingEnd="32dp"
+                    android:drawableEnd="@drawable/ic_report_breakage_down_arrow_24dp"
```

Comment:
Moved the icon here and used `android:drawableEnd` as `app:endIconDrawable` doesn't seem to have the expected color on Dark mode.

Reply:
It's fine for now, this will be replaced with our custom Component in the coming weeks.

Reply:
It's fine for now, this will be replaced with our custom Component in the coming weeks.